packages/create-spectacle/package.json

```diff
@@ -22,7 +22,7 @@
   },
   "peerDependencies": {},
   "devDependencies": {
-    "spectacle": "*",
+    "spectacle": "workspace:*",
```

Comment:
Using `workspace:*` tells `pnpm` to replace `workspace:*` with actual `spectacle` version during packaging, whereas the `"spectacle": "*"` is preserved as-is during packaging.

Reply:
Put the _actual version_ in and changesets will update the version if specified in `linked` in changsets config.

Reply:
My understanding is, if we specify a linkage between the two, then any time we do a minor version bump of `create-spectacle`, we'll also be making a minor version bump of `spectacle` (and so on). Seems like we might not want that right away.

Reply:
But as long as it works (maybe we run a quick publish experiment in the experiments repo?